game.py

```diff
@@ -18,6 +18,11 @@
 
 from math import sqrt, pi
 from random import uniform
+import threading
```

Comment:
Not sure, but... I can't see any references of threading module in the code, are we still using it?

Reply:
Oh yeah, I forgot to remove it when I discarded the previous approach. I'll remove it.